src/main/java/me/atam/atam4j/AcceptanceTestHealthCheckManager.java

```diff
@@ -1,44 +1,90 @@
 package me.atam.atam4j;
 
-import io.dropwizard.lifecycle.setup.ScheduledExecutorServiceBuilder;
 import io.dropwizard.setup.Environment;
-import me.atam.atam4j.health.AcceptanceTestsHealthCheck;
 import me.atam.atam4j.health.AcceptanceTestsState;
 
 import java.util.concurrent.TimeUnit;
 
 public class AcceptanceTestHealthCheckManager {
 
-    public static final int TEN_MINUTES_IN_SECONDS = 60 * 10;
+    private AcceptanceTestsRunnerTaskScheduler acceptanceTestsRunnerTaskScheduler;
+    private AcceptanceTestsHealthCheckInitializer acceptanceTestsHealthCheckInitializer;
 
-    private Environment environment;
-    private Class testClasses[];
-    private int testsInterval;
-
-    public AcceptanceTestHealthCheckManager(Environment environment, Class ... testClasses) {
-        this.environment = environment;
-        this.testClasses = testClasses;
-        this.testsInterval = TEN_MINUTES_IN_SECONDS;
+    public AcceptanceTestHealthCheckManager(AcceptanceTestsRunnerTaskScheduler acceptanceTestsRunnerTaskScheduler, AcceptanceTestsHealthCheckInitializer acceptanceTestsHealthCheckInitializer) {
+        this.acceptanceTestsRunnerTaskScheduler = acceptanceTestsRunnerTaskScheduler;
+        this.acceptanceTestsHealthCheckInitializer = acceptanceTestsHealthCheckInitializer;
     }
 
-    public AcceptanceTestHealthCheckManager(Environment environment, int testsInterval, Class ... testClasses) {
-        this.testsInterval = testsInterval;
-        this.environment = environment;
-        this.testClasses = testClasses;
+    public void initialise() {
+        acceptanceTestsRunnerTaskScheduler.scheduleAcceptanceTestsRunnerTask();
+        acceptanceTestsHealthCheckInitializer.initialize();
     }
 
-    public void initialise() {
-        AcceptanceTestsState acceptanceTestsState = new AcceptanceTestsState();
+    public static class AcceptanceTestsRunnerTaskSchedulerBuilder{
+
+        private AcceptanceTestsState acceptanceTestsState = new AcceptanceTestsState();
+        private Environment environment = null;
+        private Class testClasses[] = null;
+        private long initialDelay = 0;
+        private long period = 1;
```

Comment:
For ease of use, should we configure the default to 10 minutes in seconds? Just thinking of a good default which would make sense if a decent subset of use-cases..?
/cc @phillbarber